src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li key={movie.imdbId}>
+        <MovieCard movie={movie} />
+      </li>
+    ))}
+  </ul>
```

Comment:
If I remember correctly, in initial layout there was div movies with MovieCards. So, probaly would be better to stick with it
```suggestion
  <div className="movies">
    {movies.map(movie => (
        <MovieCard movie={movie} key={movie.imdbId> />
    ))}
  </div>
```

Reply:
Yes, but I think that it'll be better for semantic

Reply:
Alright then 😄